src/script/lua_api/l_noise.h

```diff
@@ -150,7 +151,8 @@ class LuaPcgRandom : public ModApiBase
 
 public:
 	LuaPcgRandom(u64 seed) : m_rnd(seed) {}
-	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed, seq) {}
+	// @TODO: set seq
+	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed) {}
```

Comment:
How about the PRNG used in core mapgen for decorations and ores, will that create different results to before? If it does that is probably acceptable as part of 0.5 breakage, the randomness of decorations and ores does not need preserving as it does not affect terrain shape.

However any changes to PRNG in `cavesRandomWalk()` will alter mgv6 terrain as the large caves have a big effect on surface terrain in that mapgen. We promise to not break or alter mgv6 as it is stable.
Do changes in `myrand` affect `cavesRandomWalk()`?